docs/ext/commands/api/exceptions.rst

```diff
@@ -186,6 +186,9 @@ Exceptions
 .. autoexception:: CommandRegistrationError
     :members:
 
+.. autoexception:: AppCommandRegistrationError
```

Comment:
this is missing in the hierarchy below

Reply:
https://github.com/DisnakeDev/disnake/pull/1107/commits/192d2b93f1c6ed5969635c9db7f13d29f880badc